views/pages/museumDetails.ejs

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
We don't need this file. The partial can just be shown on the homepage when you click on a city name.

Reply:
deleted.